WordPress/build.gradle

```
@@ -335,7 +335,7 @@ dependencies {
     implementation 'com.google.android.gms:play-services-auth:18.1.0'
     implementation 'com.android.installreferrer:installreferrer:2.2'
     implementation 'com.github.chrisbanes:PhotoView:2.3.0'
-    implementation 'org.greenrobot:eventbus:3.2.0'
+    implementation 'org.greenrobot:eventbus:3.3.0'
```

As mentioned [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2210#discussion_r764836506) they released a new version with updated proguard rules - no code changes.
```suggestion
    implementation 'org.greenrobot:eventbus:3.3.1'
```

---

cc @develric